File: 2018-x-sentimental-mario-more/mario.py
import cs50


condition = True
height = 0
while condition:
    print("Height: ")
    height = cs50.get_int()
    condition = height < 0 or height > 23

for i in range(height):
    print(" " * (height - 1 - i), end = "")
    print("#" * (i + 1), end = "")
    print("  ", end = "")
    print("#" * (i + 1), end = "\n")

File: 2017-x-mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main()
{
    int height;
    do
    {
        printf("Height: ");
        height = get_int();
    } while (height < 0 || height > 23);

    for (int i = 0; i <= height; i++)
    {
        int x = height - i;
        if (x != height)
        {
        printf("%.*s", x, "                       ");
        printf("%.*s", i, "#######################");
        printf("  ");
        printf("%.*s", i, "#######################");
        printf("\n");
        }
    }

    return 0;
}
